vega_sim/environment/environment.py

```diff
@@ -190,6 +195,10 @@ def _run(
 
         for agent in self.agents:
             agent.initialise(vega=vega)
+            if isinstance(agent, StateAgentWithWallet):
+                logging.info(
+                    f"{agent.name()} initialised with public_key = {vega.wallet.public_key(name=agent.wallet_name, key_name=agent.key_name)}"
+                )
```

Comment:
Why do we need this?

Reply:
It's a work around for not being able to connect to a wallet in console at the moment.

We can use the "View As Vega User" feature with one of the keys logged here to access all their info.